R/join_key.R

```diff
@@ -65,6 +72,12 @@ join_key <- function(dataset_1, dataset_2 = dataset_1, keys) {
     keys <- NULL
   }
 
+  parents <- if (directed && dataset_1 != dataset_2) {
+    structure(list(dataset_1), names = dataset_2)
```

Comment:
`setNames` is more precise
```suggestion
    setNames(list(dataset_1), dataset_2)
```